syndesis/src/i18n/locales/shared-translations.en.json

```diff
@@ -17,7 +17,10 @@
       "linkCreateConnection": "Create Connection",
       "linkCreateIntegration": "Create Integration",
       "Name": "Name",
+      "Pending": "Pending",
       "Published": "Published",
+      "podLogsUrl": "{{consoleUrl}}/project/{{namspace}}/browse/pods/{{podId}}?tab=logs",
+      "podEventsUrl": "{{consoleUrl}}/project/{{namespace}}/browse/pods/{{podId}}?tab=events",
```

Comment:
@elvisisking Wonder, am I abusing the localization here?  We followed this pattern in the old app, was nice to have this externalized a bit.

Reply:
If I understand the question, then yes, I think you might be misusing localization here. You are talking about the `podLogsUrl` and `podEventsUrl` keys right? These localization keys don't need any translations do they? Are you just using them for data substitution? Those 2 keys don't appear in the Italian language translations file either. And also the `viewLogs` key seems to be missing here?

Reply:
Damn, forgot `viewLogs`, thanks!

Yeah, I'll rework how these URLs are built up instead.  Though generally we may want to think about how to externalize these and also other things like the various documentation URLs that we'll eventually have to have.

Reply:
created https://github.com/syndesisio/syndesis-react-poc/issues/44 for that and sorted the above, thanks!